lxc/config/default.go

```diff
@@ -44,9 +44,3 @@ var DefaultRemotes = map[string]Remote{
 	"ubuntu":       UbuntuRemote,
 	"ubuntu-daily": UbuntuDailyRemote,
 }
-
-// DefaultConfig is the default configuration
-var DefaultConfig = Config{
-	Remotes:       DefaultRemotes,
-	DefaultRemote: "local",
-}
```

Comment:
We need to keep this one around or we're going to break most of our downstreams.